client/dashboard/sites/site/environment-switcher.tsx

```diff
@@ -184,23 +228,68 @@ const EnvironmentSwitcher = ( { site }: { site: Site } ) => {
 		stagingSiteId,
 		productionSiteId,
 		queryClient,
+		productionSite,
+	] );
+
+	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
+
+	const isStagingSiteReady =
+		isStagingSiteCreating && stagingSite && isAtomicTransferredSite( stagingSite );
+
+	useEffect( () => {
+		if ( ! stagingSite ) {
+			return;
+		}
+		if ( isStagingSiteReady ) {
+			createSuccessNotice( __( 'Staging site created.' ), { type: 'snackbar' } );
```

Comment:
Ok, I remembered incorrectly. The notification actually says `Staging site created.`, not `Staging site added.`.

But since we changed the rest of the texts to use the `add` verb, we could change it here as well. 🙂 

```suggestion
			createSuccessNotice( __( 'Staging site added.' ), { type: 'snackbar' } );
```

Additional idea: As it takes some time to get the staging site ready, we could consider making this final notification persistent - so the user needs to close it manually.

I have also created another task to add the notification after the user clicks the `+ Add staging site` button (DOTCOM-14416). But we can tackle it in a different PR. 🙂

Reply:
Well spotted! I have changed the wording and applied your suggestion of keeping the notice persistent, as I think it makes sense because is a long running task. Done as part of [acf9ae3](https://github.com/Automattic/wp-calypso/pull/105518/commits/acf9ae37b57c6c02ce473940b91acbb29a5a5d3f)